src/app/Http/Controllers/API/AuthController.php

```diff
@@ -106,4 +106,57 @@ public function user(Request $request)
     {
         return new UserResource($request->user());
     }
+  
+  /**
+     * Send an email to the user with the reset password link
+     *
+     * @bodyParam email email required email of the user who makes the request
+     *
+     * @param Request $request
+     */
+    public function sendResetLinkEmail(Request $request)
+    {
+        $request->validate([
+            'email' => 'required|email|exists:users,email'
+        ]);
+
+        // generate and save the token
+        $token = Str::random(60);
```

Comment:
Define `60` somewhere else so as to avoid magic numbers.

Reply:
there's no need of that in a php application as there are no real constant variables like in C programming and also because it's only used in one place and the code is small and readable